### ISSUE TITLE
Use setup.py for building package

### ISSUE THREAD

**Issue by SoapGentoo** [Carreau/nose_warnings_filters#4]
Dear @Carreau 
would you consider also including a **setup.py**? We would like to properly integrate _nose_warnings_filters_ in Gentoo, but using pure wheels based packages is cumbersome and in many cases unworkable for us.

**Comment by Carreau** [comment]
> Dear @Carreau 
> would you consider also including a setup.py? We would like to properly integrate nose_warnings_filters in Gentoo, but using pure wheels based packages is cumbersome and in many cases unworkable for us.

Sure; I'll look at doing that. 

You should be able to use `flit install` in the root to "replace" `pip install .`. Note that [pep 518](https://www.python.org/dev/peps/pep-0518/) is the first step toward making `setup.py` optional, I don't know if it affect you.

Do you need a classic `setup.py` that does the installation using setuptools, or is it ok for the `setup.py` to call out to flit ?

Thanks.

**Comment by SoapGentoo** [comment]
No, please use a standard `setup.py`, preferably using setuptools. Using flit is not an option, because Gentoo handles python modules slightly differently, and hence we prefer to run `setup.py` ourselves. Thanks for reacting so speedily.

**Comment by Carreau** [comment]
I should have just uploaded (and pushed on github) a version with a setup.py but it will likely not completly work yet as it does not register entrypoints. 

I'm curious to know if you have link about how gentoo handle python modules and where this is relevant.
